### PR TITLE
Bug 1168269 - Add support for Nexus 5 and replace swipe hack with res…

### PR DESCRIPTION
**Pull request by janus926**
…etting kernel cached values. r=eli

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=64ced020cf7e0c9076f21737e712038e5c87e5c4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=949e79f4dba270ffbceadae11f4ee0cdff1d9385

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b1b46906e58d4a1a7f31eaca135789227c7fa07a

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=043ee6c12a5c9d88ec38fdcad0f24f0b98f571c9

File: tools/mozdevice/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mozdevice",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (eliperelman) [inline tools/mozdevice/package.json]: Could you make this version `3.1.0`?

File: tools/mozdevice/config.json [hunk 2]
```diff
@@ -89,8 +96,7 @@
         "inputEvent": "/dev/input/event0",
         "defaultOrientation": "portrait",
         "dimensions": [480, 854],
-        "swipePadding": [40, 40, 40, 40],
-        "touchFrequency": 13
```
Comment (eliperelman) [inline tools/mozdevice/config.json]: Any reason why we are removing the touch frequency here?

File: tools/raptor/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "raptor",
-  "version": "1.6.4",
+  "version": "1.6.5",
```
Comment (eliperelman) [inline tools/raptor/package.json]: `1.7.0` please. :)

File: tools/mozdevice/config.json (hunk 2 above)
Comment (janus926) [inline tools/mozdevice/config.json]: I removed it since it's used only by swipeHack(), will keep it.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c6175e2bae3de19e7e79e88aecd370aef52fbc89

**Pull request merged**